target/linux/ar71xx/files/arch/mips/ath79/Kconfig.openwrt

```diff
@@ -1622,6 +1622,18 @@ config ATH79_MACH_TL_WR940N_V4
 	select ATH79_DEV_M25P80
 	select ATH79_DEV_WMAC
 
+config ATH79_MACH_TL_WR942N_V1
+	bool "TP-LINK TL-WR942N v1 support"
+	select SOC_QCA956X
+	select ATH79_DEV_AP9X_PCI if PCI
```

Comment:
This is not needed here, the board you are adding support for doesn't have/use pci/-e. Please, drop it from here and also clean-up mach-*.c file from related header files.